blocks/blocks.go

```diff
@@ -19,48 +20,61 @@ type Block interface {
 	Loggable() map[string]interface{}
 }
 
-// Block is a singular block of data in ipfs
-type RawBlock struct {
+type BasicBlock struct {
```

Comment:
Sorry, I missed that change as it was in a separate commit.  Let me know if you know if you want me to do anything about it, like submitting this change separately.